module-apps/apps-common/widgets/TimeFixedWidget.hpp

```diff
@@ -35,11 +41,22 @@ namespace gui
                         const std::uint32_t &h,
                         const bool minus = false);
 
+        TimeFixedWidget(Item *parent,
+                        const std::uint32_t &x,
+                        const std::uint32_t &y,
+                        const std::uint32_t &w,
+                        const std::uint32_t &h,
+                        const bool minus,
```

Comment:
Remove `const`?

Reply:
The origin ctor had const here so I did also add const here. I can remove from both ctors